src/server/migrations/sql/20190919172555-update-at-function.sql

```diff
@@ -0,0 +1,13 @@
+CREATE OR REPLACE FUNCTION updated_at() RETURNS TRIGGER AS $$
```

Comment:
ordering wise - this should be first -- just rename the date part of the filenames -- it uses that to "order" them
this needs to come before something that uses it.